### DISCUSSION TITLE
Dashboard Tab to See All Queued Builds

### DISCUSSION THREAD

**Discussion by TSIA-SN** [vercel/community#1014] [Idea]
Our team makes use of Turborepo, and there are times when we do merges and so on that it queues up builds for all Vercel projects that point to the repo. Right now if we want to see the status of all builds, we have to click into each project on separate tabs to see which ones are currently building, their current build duration, and which ones are currently waiting. It would be nice if we could have a UI where we could see all currently queued/building builds for all our Team's projects, cancel queued builds, and change the priority of builds from that same UI.

**Comment by amyegan** [comment]
Thanks for reaching out @TSIA-SN! I added this to the requested features list for the team to investigate.
